src/test/resources/net/sf/jabref/exporter/MsBibExportFormatTest1.bib

```diff
@@ -0,0 +1,44 @@
+@unpublished{,
+  author =                      {Hanns-Jörg Stoß},
+  month =                       {July},
+  note =                        {a Note},
+  abstract =                    {Es werdenTuring-Maschinen mitk unabhängigen Lese-Schreib-Köpfen auf einem Band undTuring-Maschinen mit je einem Kopf aufk Bändern betrachtet. Es wird gezeigt, daß sich ohne Verändern der Rechenzeit jede Maschine von einem Typ durch eine Maschine vom andern Typ simulieren läßt. Daraus wird gefolgert, daß es für die Rechengeschwindigkeit vonTuring-Maschinen nur auf die Zahl der Köpfe, nicht aber auf deren Verteilung auf einzelne Bänder ankommt.},
```

Comment:
In the wiki, we have at the moment 2 databases that could be used for testing.
This list could be complemented with custom-made entries fitting the purpose of the tests (for example, the listed BibTeX database does not contains an "abstract" field). This would avoid having to create entries over and over again.